slackclient/_slackrequest.py

```diff
@@ -76,4 +77,4 @@ def do(self, token, request="?", post_data=None, domain="slack.com", timeout=Non
         post_data['token'] = token
         headers = {'user-agent': self.get_user_agent()}
 
-        return requests.post(url, headers=headers, data=post_data, files=files, timeout=timeout)
+        return requests.post(url, headers=headers, data=post_data, files=files, timeout=timeout, proxies=self.proxies)
```

Comment:
This is the line tox/flake is erring on. Split this up across multiple lines and it will pass 👍